File: Software/Backend/DataAccessLayer/Models/MyJobModel.cs
namespace DataAccessLayer.Models;

/// <summary>
/// Sadrži informacije o poslu korisnika ili posllovima na koje je dodan
/// </summary>
public record MyJobModel
{
    public int Id { get; set; }
    public int EmployerId { get; set; }
    public int JobStatusId { get; set; }
    public string JobStatus { get; set; }
    public string Title { get; set; }
    public string Description { get; set; }
    public bool AllowWorkerInvite { get; set; }
    public string Location { get; set; }
    public decimal? Lat { get; set; }
    public decimal? Lng { get; set; }
    public bool UserIsEmployer { get; set; }
}

File: Software/Frontend/ws/src/main/java/hr/foi/air/baufind/ws/model/MyJobModel.kt
package hr.foi.air.baufind.ws.model

data class MyJobModel(
    val id: Int,
    val employerId: Int,
    val jobStatusId: Int,
    val jobStatus: String,
    val title: String,
    val description: String,
    val allowWorkerInvite: Boolean,
    val location: String,
    val lat: Double?,
    val lng: Double?,
    val userIsEmployer: Boolean
)